dashboard/app/models/teacher_feedback.rb

```diff
@@ -67,7 +67,17 @@ def self.get_student_level_feedback(student_id, level_id, teacher_id, script_id)
     ).latest
   end
 
-  def self.get_all_feedback_for_section(student_ids, level_ids, teacher_id)
+  # returns the latest feedback from each teacher for the student on the level
+  def self.get_latest_feedbacks_received(student_id, level_id, script_id)
+    where(
+      student_id: student_id,
+      level_id: level_id,
+      script_id: script_id
+    ).latest_per_teacher
+  end
+
+  # returns the latest feedback for each student on every level given by the teacher
+  def self.get_latest_feedbacks_given(student_ids, level_ids, teacher_id)
```

Comment:
Should usage of this method be replaced with the method on L61?  (We should always be using both script_id and level_id, right?)

Reply:
Oh! I'll add `script_id` to the query, but the difference between this and the one on line 61, is this one returns multiple feedbacks whereas the other one only returns one for one level

Reply:
Oh, is the difference that the one on 61 only returns for one student but this one returns for multiple students?  If that's the case, can we replace calls to 61 with this method (passing in an array with a single student id)?

Reply:
This one takes a list of student_ids and level_ids, I'll see what I can do!

Reply:
Updated!